README.md

```diff
@@ -3,11 +3,20 @@
 
 <br>
 
+[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=sb-exe-calismayi-durdurdu&layout=pie&theme=dark)](https://github.com/esocraton/github-readme-stats)
```

Comment:
- görselin ahenk içinde olması için en dar olan stats komponentini en yukarı aldım.